RocketChat/Rocket.Chat.ReactNative
# [REGRESSION] Clear local server cache not loading rooms

## Proposed changes

It stopped disconnecting after #2894.
That would make the app to keep session open, but without any database content.

## Issue(s)	


## How to test or reproduce

- Tap clear local server cache
- App should fetch channels again and keep stable connection

## Screenshots

## Types of changes




## Checklist



## Further comments
